signingscript/src/signingscript/sign.py

```diff
@@ -1306,7 +1309,15 @@ def verify_mar_signature(cert_type, fmt, mar, keyid=None):
     else:
         crosscert = None
 
-    if not await winsign.sign.sign_file(infile, outfile, digest_algo, certs, signer, url=url, crosscert=crosscert, timestamp_style=timestamp_style):
+    if authenticode_comment and orig_path.endswith(".msi"):
+        log.info("Using comment '%s' to sign %s", authenticode_comment, orig_path)
+    elif authenticode_comment:
+        log.info("Not using specified comment to sign %s, not yet implemented.", orig_path)
```

Comment:
can you change this to say "not yet implemented for non-msi files" or something like that?